### PR TITLE
Disable v1alpha3 from pre-submit on circle and prow

### PR DESCRIPTION
**Pull request by kyessenov**
Signed-off-by: Kuat Yessenov <kuat@google.com>

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -384,7 +384,7 @@ jobs:
             make docker.all generate_yaml
       - run: bin/testEnvRootMinikube.sh wait
       - run: docker images
-      - run: make e2e_pilot HUB="${HUB}" TAG="${TAG}" TESTOPTS="--skip-cleanup -mixer=true -auth_enable=true -v1alpha3=true -v1alpha1=true -errorlogsdir=/home/circleci/logs -use-sidecar-injector=false --core-files-dir=/home/circleci/logs"
```
Comment (costinm) [inline .circleci/config.yml]: this is not required already - runs only in periodic.

You want to change e2e-pilot job, and use "make test/minikube/noauth/e2e_pilot_alpha1"

Comment (kyessenov) [reply .circleci/config.yml]: Done

**Comment by sebastienvas** [comment]
why are we doing this ?

**Comment by kyessenov** [comment]
prow v1alpha3=true flaked on this non-functional PR.
circle v1alpha3=true flaked on https://github.com/istio/istio/pull/4019

**Comment by kyessenov** [comment]
Both my and @ijsnellf assessment is that it's too early for v1alpha3 to be run in pre-submit. It's run as post-submit.

**Comment by kyessenov** [comment]
And it flaked again... The flag is non-effective.

**Comment by ldemailly** [comment]
pilot/pkg/proxy/envoy/v2/eds.go:253:6:error: func connectionId should be connectionID (golint)

File: .circleci/config.yml [hunk 2]
```diff
@@ -223,7 +223,7 @@ jobs:
             make docker.all generate_yaml
       - run: bin/testEnvRootMinikube.sh wait
       - run: docker images
-      - run: make test/minikube/noauth/e2e_pilot HUB="${HUB}" TAG="${TAG}"
+      - run: make test/minikube/auth/e2e_pilot_alpha1 HUB="${HUB}" TAG="${TAG}"
```
Comment (costinm) [inline .circleci/config.yml]: Why change it to auth ? That's still pretty flaky. Please keep the auth separated.

Comment (kyessenov) [reply .circleci/config.yml]: There's no script to run v1alpha1 without auth

**Comment by kyessenov** [comment]
Force merging to unblock the build.

**Comment by ldemailly** [comment]
I guess you could also have changed the required/not required bit on github settings for the branch?

**Pull request merged**